test-suite/src/foreign_key.rs

```diff
@@ -148,7 +148,7 @@ test_case!(foreign_key, {
     .await;
 
     g.named_test(
-        "Creating table with foreign key should be succeeded if referenced table has primary key. NO ACTION(=RESTRICT) is default",
+        "Creating table with foreign key should succeeded if referenced table has primary key. NO ACTION(=RESTRICT) is default",
```

Comment:
could you make a separate PR for this? then I can merge right after.
It looks not related to this PR so..

and
"should be succeeded" -> "should succeeded" -> "should succeed" ?